omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ConceptMapController1_8Test.java

```diff
@@ -1,108 +0,0 @@
-/**
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
- * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
- * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
- * graphic logo is a trademark of OpenMRS Inc.
- */
-package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs1_8;
-
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThat;
-
-import java.util.UUID;
-
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.openmrs.Concept;
-import org.openmrs.ConceptMap;
-import org.openmrs.api.ConceptService;
-import org.openmrs.api.context.Context;
-import org.openmrs.module.webservices.rest.SimpleObject;
-import org.openmrs.module.webservices.rest.test.OpenmrsProfileRule;
-import org.openmrs.module.webservices.rest.web.RestTestConstants1_8;
-import org.openmrs.module.webservices.rest.web.api.RestHelperService;
-import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
-import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
-
-/**
- * Tests CRUD operations for {@link ConceptMapType}s via web service calls
- */
-public class ConceptMapController1_8Test extends MainResourceControllerTest {
```

Comment:
Shouldn't this have been migrated?

Reply:
noted 
will correct that

Reply:
Hello @ibacher 
the `ConceptMapController1_8Test` is it the same as testing the `ConceptMapController1_9Test` which the same methods thats is why didn't include it 
the solution i had was to merge them to one test case but i realized they are testing the same methods 
what's your take on this

Reply:
That's fine then. As long as the tests are covered.